force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -162,17 +208,22 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static String repeatString(String inputString, Integer repeatCount) {
         // Initialize the result String
-
         // Use a for loop to append the inputString to the result string repeatCount times
         /*
         for () {
-
             // If it is not the last iteration, add a semicolon to separate the strings
         }
         */
-
         // Return the final result string
-        return null; // Replace null with the variable you used to store the result
+        String result = '';
+        for(Integer i = 0; i < repeatCount; i++){
+            result += inputString;
+            if(i < repeatCount - 1){
+                result += '; ';
+            }
+        }
+        return result; // Replace null with the variable you used to store the result
```

Comment:
Soon you'll find better ways to do this that are a bit simpler and don't require you to write the loop. :)